coordinator/migrations/2023-09-04-090152_add_order_state_and_matches/up.sql

```diff
@@ -0,0 +1,12 @@
+-- Your SQL goes here
+CREATE TYPE "OrderState_Type" AS ENUM (
+    'Open',
+    'Match',
+    'Taken',
+    'Failed'
+);
+
+ALTER TABLE "orders" ADD COLUMN "order_state" "OrderState_Type" NOT NULL DEFAULT 'Open';
+
+UPDATE orders SET order_state = 'Taken' WHERE taken = true;
+UPDATE orders SET order_state = 'Failed' WHERE taken = false;
```

Comment:
Note: when running this the first time, all open not taken orders will be set to 'failed'. Not an issue because our maker will create new orders immediately, but I wanted to point it out :)

Reply:
I'll exclude limit orders